add links between CMEK management and reference pages (#14372)
cockroachdb/docs: cockroachcloud/managing-cmek.md

docs_area: manage.security
---

> [Customer-Managed Encryption Keys (CMEK)](cmek.html) for {{ site.data.products.dedicated }} allow the customer to delegate responsibility for the work of encrypting their cluster data to {{ site.data.products.dedicated }}, while maintaining the ability to completely revoke {{ site.data.products.dedicated }}'s access.

Reviewer: I prefer the previous version where we are talking about CMEK the feature rather than CMEK the key.

{% include cockroachcloud/cockroachcloud-ask-admin.md %}

> This page guides the user through the process of enabling [Customer-Managed Encryption Keys (CMEK)](cmek.html) for your {{ site.data.products.dedicated }} cluster.

Reviewer: ```suggestion
This page shows how to enable [Customer-Managed Encryption Keys (CMEK)](cmek.html) for a {{ site.data.products.dedicated }} cluster.
```
Rewriting for second person.
